idnc/estimation/estimate_target_position.py

```diff
@@ -32,7 +33,7 @@ def from_txt_label_to_array(
 
 
 def labels_to_relative_angles(
-    coordinates: 'dict[str, float]',
+    coordinates: "dict[str, float]",

```

Comment:
Use Dict from typing for better type hint

Reply:
Resolved in the meantime (I think)